### PR TITLE
Rename asn1 ber dependency.

### PR DESCRIPTION
**Pull request by vagruchi**

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -1,2 +1,4 @@
-gopkg.in/asn1-ber.v1 v1.0.0-20181015200546-f715ec2f112d h1:TxyelI5cVkbREznMhfzycHdkp5cLA7DpE+GKjSslYhM=
-gopkg.in/asn1-ber.v1 v1.0.0-20181015200546-f715ec2f112d/go.mod h1:cuepJuh7vyXfUyUwEgHQXw849cJrilpS5NeIjOWESAw=
+github.com/go-asn1-ber/asn1-ber v1.3.1 h1:gvPdv/Hr++TRFCl0UbPFHC54P9N9jgsRPnmnr419Uck=
+github.com/go-asn1-ber/asn1-ber v1.3.1/go.mod h1:hEBeB/ic+5LoWskz+yKT7vGhhPYkProFKoKdwZRWMe0=
+github.com/go-asn1-ber/asn1-ber v1.0.0-20181015200546-f715ec2f112d h1:TxyelI5cVkbREznMhfzycHdkp5cLA7DpE+GKjSslYhM=
+github.com/go-asn1-ber/asn1-ber v1.0.0-20181015200546-f715ec2f112d/go.mod h1:cuepJuh7vyXfUyUwEgHQXw849cJrilpS5NeIjOWESAw=
```
Comment (johnweldon) [inline go.sum]: Is it possible to remove these two lines, or is `go mod tidy` pulling them in again?

Comment (vagruchi) [reply go.sum]: My bad. Forgot to use go mod tidy.

**Comment by johnweldon** [comment]
Need to fix travis first I think

**Comment by vagruchi** [comment]
Fixed Travis. Not sure should I change go_import_path or not?
